saleor/graphql/account/mutations/base.py

```diff
@@ -330,8 +330,9 @@ def perform_mutation(cls, _root, info, **data):
         cls._save_m2m(info, address, cleaned_input)
 
         user = address.user_addresses.first()
-        user.search_document = prepare_user_search_document_value(user)
-        user.save(update_fields=["search_document", "updated_at"])
+        if user:
+            user.search_document = prepare_user_search_document_value(user)
+            user.save(update_fields=["search_document", "updated_at"])
         manager = get_plugin_manager_promise(info.context).get()
         cls.call_event(manager.customer_updated, user)
```

Comment:
Is it a reason to call `customer_updated` in case of missing `user`?

Reply:
No, but it's removed anyway in #11395